tests/unit/test_ansible_inventory.py

```diff
@@ -118,3 +129,64 @@ def test_invalid_layouts(self, layout, db, metadata):
         with pytest.raises(ConfigError) as excinfo:
             ans_inv.create_inventory()
         assert "dictionary" in str(excinfo.value)
+
+    @pytest.mark.parametrize(
+        "layout",
+        [
+            # It is more tolerant with falsy values
+            common_inventory_layout(),
+            None,
+            {},
+            empty_layout(),
+            [],
+            False,
+        ],
+    )
+    def test_meta_extra(self, layout, db_meta_extra, metadata):
+
+        config = provisioning_config(layout)
+        ans_inv = AnsibleInventoryOutput(config, db_meta_extra, metadata)
+        inventory = ans_inv.create_inventory()
+
+        first_host = inventory["all"]["hosts"][
```

Comment:
think you can also use e.g.
```python
first_host = inventory["all"]["hosts"].values()[0]
```
which seems easier/more readable (untested though)

Reply:
I also have observed that inventory output doesn't output host in the same order as in metadata.

Reply:
It probably doesn't matter much.  All the host should get the attr - in current mock data preparation.  Anyway, this looks also fine.